backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1712,3 +1732,51 @@ class WorkspaceSubscriptionSchema(marshmallow.Schema):
         format=DATETIME_FORMAT, description="evaluation date", allow_none=True
     )
     evaluator = marshmallow.fields.Nested(UserDigestSchema(), allow_none=True)
+
+
+class UserIdSchema(marshmallow.Schema):
+    """
+    Simple user id schema
+    """
+
+    user_id = marshmallow.fields.Int(example=3, required=True)
+
+
+class GetUserFollowQuerySchema(BasePaginatedQuerySchema):
+    """Possible query parameters for the GET following and followers endpoint."""
+
+    user_id = marshmallow.fields.Int(
+        example=42, validate=strictly_positive_int_validator, allow_none=True, default=None
+    )
+
+    @post_load
+    def user_follow_query(self, data: typing.Dict[str, typing.Any]) -> UserFollowQuery:
+        return UserFollowQuery(**data)
+
+
+class UserFollowSchemaPage(BasePaginatedSchemaPage):
+    items = marshmallow.fields.Nested(UserIdSchema, many=True)
+
+
+class DeleteUserFollowPathSchema(UserIdPathSchema):
+    user_id_2 = marshmallow.fields.Int(
```

Comment:
Maybe `followed_user_id`?

Reply:
`user_id_2` is in the specs. @grignards can i change it or this implies some other changes ?

Reply:
<user_id_1> and <user_id_2> are placeholders in the issue, not the naming to use (but I now see this wasn't explicit enough).

Reply:
I will use `leader_user_id` to have the same name than database and model.

Reply:
done